pyproject.toml

```diff
@@ -98,7 +110,7 @@ sphinx = { version = "<7.2", optional = true }
 sphinx-design = { version = "*", optional = true }
 sphinxcontrib-autoprogram = { version = "*", optional = true }
 m2r2 = { version = "*", optional = true }
-brassy = { version = "*", optional = true }
+brassy = { version = ">=0.0.3", optional = true }
```

Comment:
@biosafetylvl5 We have been using brassy long enough here that someone's stuff broke becaause they needed to update brassy for build_docs.py!  Definitely need >=0.0.3

Reply:
Oh wow, that's fun :) Yes, I think brassy >=0.0.3 is good 👍